.changeset/eleven-goats-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/autocomplete": patch
+---
+
+Added missing AsyncFiltering story in Autocomplete
```

Comment:
@wingkwong as this change only impacts the story we don't need to add a changeset

Reply:
Updated.